packages/frontend/src/features/reporting/hooks/useReputationScore/useReputationScore.ts

```diff
@@ -1,11 +1,22 @@
 import { QueryKeys } from '@/constants/queryKeys'
+import { useUserState } from '@/features/core'
 import { useQuery } from '@tanstack/react-query'
 
 export function useReputationScore() {
-    return useQuery({
+    const { getGuaranteedUserState } = useUserState()
+    const { data: reputationScore } = useQuery({
         queryKey: [QueryKeys.ReputationScore],
         queryFn: async () => {
-            return -1
+            const userState = await getGuaranteedUserState()
+            const data = await userState.getData()
+            return Number(data[0].toString()) + 1
```

Comment:
I think reputation score should be calculated as `data[0] - data[1]` because `data[0] is positive reputation and `data[1]` is negative reputation. We can ask @vivianjeng to confirm the calculation.

You can only use `Number()` to cast bigint value. For example, `Number(1n)`